test/config/hosted.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version='1.0' encoding='UTF-8'?>
 <library>
   <modules>
-    <module>:platform:core</module>
-    <module>:test:**</module>
+    <module>modm:platform:core</module>
+    <module>modm:test:**</module>
```

Comment:
The tests have all been moved to the modm-test repository in #110.

Reply:
I've updated the commit.